### DISCUSSION TITLE
BookingWP add-on 🔥

### DISCUSSION THREAD

**Discussion by Uhtredtb** [inspireui/support#5552] [New Feature/Ideas Requests]
Please support BookingWP add-on this is a major thing to make the appointment app fully integrate to your app. With full functions.

@trunghieuvn @minhcasi

**Comment by bakpaorabbit** [comment]
what I know bookingWP is built in WooCommerce Product Add-On. the product add on not working in extended version ?

**Reply by Uhtredtb** [reply]
@bakpaorabbit They do not support yet however, you can help by voting this feature. So that I can happens.

**Reply by bakpaorabbit** [reply]
since I don't use extended license now, I never test this feature, but if add on booking wp not available since it has same plugin (built in) with woocommerce product add on, +1 for this !! @minhcasi

**Reply by bakpaorabbit** [reply]
@bambootb question for you. Simple product with add ons show in your app ?, I mean a product with bookingwp add ons (not appointment product) since it will disabled original woocommerce product add ons.

**Reply by Uhtredtb** [reply]
@bakpaorabbit I don't see any Woocommerce Product add-ons show whether an appointment product or simple product. It doesn't seem to be support yet?

**Comment by davidom0101** [comment]
+1 for this feature
Badly needed
